lib/network/http/utils.js

```diff
@@ -104,6 +105,20 @@ function parseRange(rangeHeader, objectSize) {
     return getByteRangeFromSpec(rangeSpec, objectSize);
 }
 
-module.exports = { parseRangeSpec,
+function checkSupportIPv6() {
+    const niList = os.networkInterfaces();
+    let supportIPv6 = false;
+    Object.keys(niList).forEach(network => {
```

Comment:
the `forEach` will overwrite `supportIPv6`, so only the last network will be checked. Maybe another `.some`?

Reply:
that's a good point

Reply:
fixed 😸